games/service_igdb.py

```diff
@@ -0,0 +1,191 @@
+import requests
+from requests.structures import CaseInsensitiveDict
+import json
+import re
+import environ
+
+
+env = environ.Env()
+environ.Env.read_env()
+
+
+class MyAuth:
+    """
+    Main goal is get token, in order to dos, we need credentilas(client_id, secret
+    """
+
+    def __init__(self, secret: str, client_id: str, url: str):
+        self.secret = secret
+        self.client_id = client_id
+        self.url = url
+
+    def get_token_twitch(self, grant_type: dict):
+        """
+        1. Create post request
+        2. Provide cretentials and grant_type(applicable for twitch
+        """
+        auth_body = {
+            "client_id": self.client_id,
+            "client_secret": self.secret,
+        }
+        auth_body = auth_body | grant_type
+        auth_response = requests.post(self.url, auth_body)
+        return auth_response.json()
+
+    def get_token_twitter(self, hash: str):
+        headers = CaseInsensitiveDict()
+        headers["Content-Type"] = "application/x-www-form-urlencoded"
+        headers["Authorization"] = f"Basic {hash}"
+        data = "grant_type=client_credentials"
+        auth_response = requests.post(URL_TWITTER_AUTH, headers=headers, data=data)
+        return auth_response
+
+
+# igdb
+URL_TWITCH = "https://id.twitch.tv/oauth2/token"
+client_secret_twitch = env("client_secret_twitch")
+client_id_twitch = env("client_id_twitch")
```

Comment:
Could you please move these variables into settings.py also? All env vars should be there